pyproject.toml

```diff
@@ -138,19 +138,16 @@ ignore = [
     # Temporary ignored due to API change
     "PTH", # Use pathlib.Path
 
+    "COM812", # Trailing comma missing
```

Comment:
why did you move this from the original diff?

this is still in starbase, so keeping it below seems appropriate

Reply:
I will remove the ignored rules below in the following PRs that fix them, so I put this one on top that should not be removed.